lib/builder.js

```diff
@@ -383,6 +387,11 @@ module.exports = class Builder extends EventEmitter {
     // Why?
     reparentNodes(outputNodeWrapper);
 
+    // During cleaning, heimdall isn't structured right and aggregateTime() causes an OOM
+    if (this._cancelationRequest.isCanceled) {
```

Comment:
This shouldn't be necessary if this really fixes the issue.

Reply:
you are correct!